bin/test

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
```

Comment:
Would be great if we could keep this top level `bin/test` even if it's not used for Docker. It's useful to run locally.

Reply:
@davidrapson Are you happy with the changes?

Reply:
Just tried this out locally and words great. Lovely stuff